docs/providers/aws/events/cloudwatch-event.md

```diff
@@ -90,3 +90,24 @@ functions:
                 - pending
           inputPath: '$.stageVariables'
 ```
+
+## Specify Input or Inputpath
```

Comment:
Could you please re-phrase this headline 😬

Reply:
🤐